enterprise/replicasync/replicasync.go

```diff
@@ -408,9 +408,6 @@ func (m *Manager) AllPrimary() []database.Replica {
 			continue
 		}
 
-		// When we assign the non-pointer to a
-		// variable it loses the reference.
-		replica := replica
```

Comment:
This is the only instance I'm slightly wary of, but the new behaviour seems to be as intended here. It was originally added in #4596 to avoid a test race.